### DISCUSSION TITLE
Is there any way to prevent paragraphs from highlighting every time I scroll the text?

### DISCUSSION THREAD

**Discussion by Pretinaverse** [edrlab/thorium-reader#2514] [Q&A] [answered]
Every time I scroll the text, the first or second paragraph in view is highlighted. I get the feeling that it has something to do with the TTS function, but I don't really care about the TTS function and it's distracting to have the text constantly highlighting. Is there any way to disable this function?

![Sin título](https://github.com/user-attachments/assets/96813fb2-f718-4945-9a33-2be74a38abd0)

**Comment by danielweck** [comment, marked as answer]
Hello, this visual outline indicates the "current reading location" and I agree that it is a distracting feature when the user just scrolls the content viewport or turns pages. However it is a very useful feature when hyperlinking into a specific spot inside a page spread or somewhere precise inside the content extent of the scrollview. I will examine how to disable the feature in the annoying case, while keeping it active in the useful case :) Thank you for reporting your feedback!

**Comment by danielweck** [comment]
Let's continue the discussion in the actual issue: https://github.com/edrlab/thorium-reader/issues/2528
